.github/workflows/pr.yaml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     services:
       a3p:
-        image: ghcr.io/agoric/agoric-3-proposals:latest
+        image: ghcr.io/agoric/agoric-3-proposals@sha256:644e09bf041e5588570ff9a49e6129cc468e9c8458417541027e8532cb91083b
```

Comment:
Ideally it works with latest but we don't have CI set up yet to trigger here when `latest` changes. Meanwhile it should at least use a tag.

Does [use-upgrade-17](https://github.com/Agoric/agoric-3-proposals/pkgs/container/agoric-3-proposals/320174554?tag=use-upgrade-17) work? That's the current latest.